### DISCUSSION TITLE
plotting e11 map

### DISCUSSION THREAD

**Discussion by Patcurran** [MechMicroMan/DefDAP#110] [General]
Hi Everyone,

how do you plot the e11 strain map?

best wishes,
Patrick

**Comment by kata-neumann** [comment]
hello,
how did you figure it out? I am interested in a similar thing. In the final plotting part: 
plot = dicMap.plotMaxShear(
    vmin=0.0, vmax=0.1, plotScaleBar=True,
    plotGBs=True, dilateBoundaries=True
)
but I want to see only e11.

**Comment by davidlunt88** [comment]
from defdap.plotting import MapPlot

mapData = dicMap.e11
mapData = dicMap.crop(mapData)

plot = MapPlot.create(
    dicMap, mapData,
    vmin=0, vmax=0.08, plotColourBar=True, cmap="viridis", plotScaleBar=True
)

From: kata-neumann ***@***.***>
Date: Wednesday, 3 April 2024 at 13:07
To: MechMicroMan/DefDAP ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [MechMicroMan/DefDAP] plotting e11 map (Discussion #110)

hello,
how did you figure it out? I am interested in a similar thing. In the final plotting part:
plot = dicMap.plotMaxShear(
vmin=0.0, vmax=0.1, plotScaleBar=True,
plotGBs=True, dilateBoundaries=True
)
but I want to see only e11.
